lib-multisrc/gmanga/src/eu/kanade/tachiyomi/multisrc/gmanga/Dto.kt

```diff
@@ -123,7 +124,7 @@ class NameDto(val name: String)
 @Serializable
 class TypeDto(
     val name: String,
-    val title: String,
+    val title: String?,
```

Comment:
also check for nullability when adding to genres, so "null" doesn't end up in the genre string